crates/ingress-http/Cargo.toml

```diff
@@ -19,9 +19,14 @@ restate-ingress-dispatcher = { workspace = true }
 restate-schema-api = { workspace = true, features = ["component"]}
 restate-types = { workspace = true }
 
+# To invoke the awakeables built-in service
+restate-pb = { workspace = true }
+prost = { workspace = true }
+
 # Encoding/Decoding
 bytes = { workspace = true }
 serde = { workspace = true }
+serde_with = { workspace = true }
 serde_json = { workspace = true }
```

Comment:
nit: I think we are no longer trying to group dependencies logically because it increases the maintenance burden and there is sometimes no clear grouping. What we try to do is to group restate dependencies and others together.